packages/driver/src/cypress/runner.ts

```diff
@@ -147,11 +149,14 @@ const setWallClockDuration = (test) => {
 // tests to an id-based object which prevents
 // us from recursively iterating through every
 // parent since we could just return the found test
-const wrap = (runnable) => {
+const wrap = (runnable): Record<string, any> | null => {
   return $utils.reduceProps(runnable, RUNNABLE_PROPS)
 }
 
-const wrapAll = (runnable): any => {
+// Reduce runnable down to its props and collections.
+// Sent the the Reporter to populate command log
```

Comment:
Sent by who? I'm not really sure what this comment is trying to tell me.

Edit: I think it makes more sense below where wrapAll is used, rather than up here where it's defined.